html/css/dark.css

```diff
@@ -7213,12 +7213,26 @@
   .expandable:hover span {
       background-color: #2e3338;
   }
-  table.table tr:not(:first-child){
```

Comment:
Could we add the class to the tables that's missing it instead?

Reply:
The code to set this table a class is buried inside lots of confusing php files... i could also do it, but we face the risk to not find all the use scenarios/cases in which the table is used.